Description: Force openmpi ext2x to work with pmix version 3
 Based on a hint from rhc@open-mpi.org
Author: Alastair McKinstry <mckinstry@debian.org>
Last-Updated: 2018-07-14
Forwarded: not-needed


Index: openmpi-3.1.1.real/opal/mca/pmix/ext2x/configure.m4
===================================================================
--- openmpi-3.1.1.real.orig/opal/mca/pmix/ext2x/configure.m4
+++ openmpi-3.1.1.real/opal/mca/pmix/ext2x/configure.m4
@@ -32,8 +32,8 @@ AC_DEFUN([MCA_opal_pmix_ext2x_CONFIG],[
 
     AS_IF([test "$opal_external_pmix_happy" = "yes"],
           [ # check for the 2.x version
-           AC_MSG_CHECKING([if external component is version 2.x])
-           AS_IF([test "$opal_external_pmix_version" = "2x"],
+           AC_MSG_CHECKING([if external component is version 3.x])
+           AS_IF([test "$opal_external_pmix_version" = "3x"],
                  [AC_MSG_RESULT([yes])
                   AS_IF([test "$opal_event_external_support" != "yes"],
                         [AC_MSG_WARN([EXTERNAL PMIX SUPPORT REQUIRES USE OF EXTERNAL LIBEVENT])
